docs/guides/auth.md

```diff
@@ -180,7 +180,7 @@ var client = new Greeter.GreeterClient(channel);
 ####Base case - No encryption/authentication
```

Comment:
@LisaFC 
after this PR is merged. I'll do a pass through the doc to remove all the:
 ####Base case - No encryption/authentication
which are under
###Authenticating with Google (language X).

These sections do not  make much sense.